src/evo/deterministicmns.h

```diff
@@ -62,30 +62,47 @@ class CDeterministicMN
     uint256 proTxHash;
     COutPoint collateralOutpoint;
     uint16_t nOperatorReward{0};
+    uint16_t nType{MnType::Regular.index};
```

Comment:
why index is `uint8_t` but here is uint16_t? btw, better to use `enum class` here as more error prune (see other related comment)